src/rateLimit.ts

```diff
@@ -0,0 +1,17 @@
+import rateLimit from 'express-rate-limit';
+import { rpcError, sha256 } from './utils';
+
+const whitelistKeyHashes = ['a67acacdb3b382847fc058c206d6290c9873ccea4c656e06f2242d573a553db8'];
+
+export default rateLimit({
+  windowMs: 16 * 1e3,
```

Comment:
Need to use https://github.com/snapshot-labs/snapshot-hub/blob/master/src/helpers/rateLimit.ts#L8 otherwise we wont have the proper IP